### PR TITLE
Add Logitech Media Server to 12.2-RELEASE

### PR DESCRIPTION
**Pull request by titou386**

### CODE REVIEW

**Comment by titou386** [comment]
HI,
I'm still on FreeNAS 11.3.
How can I make this plugin available on both FreeNAS & TrueNAS ? I'm pretty sure that it is compatible TrueNAS without modification.
Thanks.

**Comment by fulder** [comment]
> How can I make this plugin available on both FreeNAS & TrueNAS ? I'm pretty sure that it is compatible TrueNAS without modification.

Should be enough to port this PR/plugin/master to the `11.3-RELEASE` branch. 

But do you still need to run the plugin in the (old) 11.3 FreeBSD version? Or wouldn't it work to run it using the currently supported 12.2 for both FreeNAS and TrueNAS?

File: logitechmediaserver.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    "p5-Crypt-OpenSSL-RSA",
+    "p5-IO-Socket-SSL",
+    "bash",
+    "curl",
+    "flac",
+    "faad2",
+    "lame",
+    "sox"
+    ],
+  "packagesite": "http://pkg.FreeBSD.org/FreeBSD:11:amd64/quarterly",
```
Comment (fulder) [inline logitechmediaserver.json:24]: Could this perhaps be changed to `http://pkg.FreeBSD.org/${ABI}/latest` to use the `latest` packagesite and make it more dynamic?

Comment (titou386) [reply logitechmediaserver.json]: > Should be enough to port this PR/plugin/master to the `11.3-RELEASE` branch.

Oohh right, I didn't pay attention to the branches.



> Could this perhaps be changed to `http://pkg.FreeBSD.org/${ABI}/latest` to use the `latest` packagesite and make it more dynamic?

I will modify.

**Comment by titou386** [comment]
Works on TrueNAS-12.0-U2.1
I'll make a PR to the 11.3-RELEASE branch

**Review by fulder — APPROVED** [review]
Thanks for the changes :) Adding my approval

**Review by sonicaj — APPROVED** [review]
Thank you @titou386 for your contribution

**Pull request merged**